src/common/components/table-pk-picker/table-pk-picker.component.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classes from './table-pk-picker.component.module.css';
+import { FieldTree } from './components/field-tree.component';
+import { ExpandDown } from '../icons/expand-down-icon.component';
+import { OptionVm } from './table-pk-picker.model';
+import { GUID } from '@/core/model';
+
+interface Props {
+  name: string;
+  options: OptionVm[];
+  label?: string;
+  selectedKeyFieldId?: GUID;
+  selectTitle?: string;
+  onKeySelected: (field: OptionVm) => void;
+}
+
+export const TablePkPicker: React.FC<Props> = props => {
```

Comment:
The name of this one is fine since is specific case to pick a table

Reply:
Maybe instead of common we will place it under relation but right now we can skip it, we will do some refactor in later steps

Reply:
Ok